src/test/ui/match/match-arm-resolving-to-never.stderr

```diff
@@ -5,17 +5,20 @@ LL | /     match E::F {
 LL | |         E::A => 1,
 LL | |         E::B => 2,
 LL | |         E::C => 3,
-LL | |         E::D => 4,
-LL | |         E::E => unimplemented!(""),
-   | |                 ------------------ this and all prior arms are found to be of type `{integer}`
+...  |
 LL | |         E::F => "",
    | |                 ^^ expected integer, found reference
 LL | |     };
    | |_____- `match` arms have incompatible types
+   | 
+  ::: $SRC_DIR/libstd/macros.rs:LL:COL
+   |
+LL | /         $crate::rt::begin_panic_fmt(&$crate::format_args!($fmt, $($arg)+),
```

Comment:
e.g. this is exposing `std::rt` as a module and this function which are intentionally private and unstable...

Reply:
Agreed on this one for sure tho. :)